website/docs/Basic_operations/update.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 4
+---
+
+
+# Update operation
+
+The update operation modifies a document record in a collection, based on a given query parameter and update.
+The `$set` operator assigns the updated record to the document.
+
+## Modify a single document
+
+Use the `updateOne()` method to update a single document in a collection.
+This operation filters a collection using a query parameter, and updates given fields within that document.
+
+```sh
+db.collection.updateOne({<query-params>}, {$set: {<update fields>}})
+```
+
+First, populate the database with a collection containing a list of documents.
+
+```sh
+db.scientists.insertMany([{firstname: "Thomas", lastname: "Edison", born: 1847, invention: "LightBulb", nobel:true},{firstname: "Graham", lastname: "Bell", born: 1847, invention: "telephone", nobel:false},{firstname: "Nikola", lastname: "Tesla", born: 1856, invention: "Tesla coil", nobel:false}, {firstname: "Ada", lastname: "Lovelace", born: 1815, invention: "Computer programming", nobel:false}])
+```
+
+Using the document record in the collection, update the document where name is `firstname` is "Graham", and set the `firstname` as "Alexander Graham".
```

Comment:
I don't get this sentence :) "where name is `firstname` is..." I think there is a typo somewhere.